includes/class-wsu-syndicate-shortcode-people.php

```diff
@@ -508,6 +522,21 @@ private function generate_item_html( $person, $type, $atts ) {
 
 			return $html;
 
+		} elseif ( 'card' === $type ) {
+
+			$heading_tag = ( ! empty( $atts['heading_tag'] ) ) ? $atts['heading_tag'] : 'h3';
+
+			$opening_heading_tag = '<' . esc_html($heading_tag) . ' class="wsu-c-card__heading">';
+			$closing_heading_tag = '</' . esc_html($heading_tag) . '>';
+
```

Comment:
These should be escaped when outputted (in the template) not when being concatenated.